README.md

```diff
@@ -1,19 +1,56 @@
-# Lizardboard
+# 🐉 DragonBoard-Backend 🐲
 
 An open source clone of [geckoboard](https://www.geckoboard.com/). Interested in contributing? Take a look at the [project page](https://github.com/GuildCrafts/lizardboard/projects) and [issues page](https://github.com/GuildCrafts/lizardboard/issues) for outstanding issues.
 
-## Contributing to Lizardboard
+## Contributing to DragonBoard
 
 Please read the [contribution guidelines](https://github.com/GuildCrafts/lizardboard/blob/master/CONTRIBUTING.md)
 
-## Setting up Development Environment
+## Set up Development Environment with Homebrew
+```
+$ brew update
+```
 
-A mongodb database named lizardboard must be created prior to starting the application for the first time.
-- brew install yarn
-- brew install mongodb
-- Ensure `mongo` is running
-- yarn install
+### Install Yarn 🐈
+```
+$ brew install yarn
+```
+### Install Mongo
+```
+$ brew install mongodb
+$ brew services start mongodb
+```
 
+###### Create db directory
```

Comment:
I don't think these steps are necessary if mongo is installed via homebrew? (This one through `Run mongo daemon`.)